lib/src/test/java/net/fushizen/invokedynamic/proxy/Utils.java

```diff
@@ -0,0 +1,9 @@
+package net.fushizen.invokedynamic.proxy;
+
+import org.junit.Assume;
+
+public class Utils {
+    public static void ignoreOnJava9() {
+        Assume.assumeFalse(System.getProperty("java.vm.specification.version").equals("9"));
```

Comment:
Since we (presumably) expect this to be broken in Java 10 as well, this should probably be either a greater-than, or an assumeTrue(version == 8) check.